api/sys_auth.go

```diff
@@ -12,6 +12,39 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
+func (c *Sys) GetAuth(path string) (*AuthMount, error) {
+	return c.GetAuthWithContext(context.Background(), path)
+}
```

Comment:
Do we need this one if we have the one below? From my experience, usually we add the `*WithContext` and keep the one method without `*WithContext` for backwards compatibility, but since these are both new, I think we could get away with just having `GetAuthWithContext`.

With that said, the Vault conventions could be different.

Reply:
I had the same question but kept it for consistency.